modules/installation-aws-iam-policies-about.adoc

```diff
@@ -8,6 +8,11 @@
 
 By default, the installation program creates instance profiles for the bootstrap, control plane, and compute instances with the necessary permissions for the cluster to operate.
 
+[NOTE]
+====
+To enable pulling images from Amazon Elastic Container Registry (ECR) postinstall in a {sno} cluster, you must add the `AmazonEC2ContainerRegistryReadOnly` policy to the IAM role associated with the cluster's control plane role.
```

Comment:
```suggestion
To enable pulling images from the Amazon Elastic Container Registry (ECR) as a postinstallation task in a {sno} cluster, you must add the `AmazonEC2ContainerRegistryReadOnly` policy to the IAM role associated with the cluster's control plane role.
```

control plane role/control plane node ?

Reply:
I think it's role in this instance, as in single-node Openshift there's only one node that contains control plane and worker functions.